src/test/common/utils/isCCBCCaseReference.ts

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-expression */
+import { expect } from 'chai'
+import { isCCBCCaseReference } from 'common/utils/isCCBCCaseReference'
+
+describe('isCCBCCasePrefix', () => {
+  it('should return true if it is a prefix', () => {
+    const referenceNumber = 'AA131231'
+    expect(isCCBCCaseReference(referenceNumber)).to.be.true
+  })
+
+  it('should return true when prefix is contained but not at the start', () => {
```

Comment:
Maybe this is just my thing but for me prefix means something placed at the beginning. Shall we change the wording to say 'identifier' everywhere here instead of 'prefix'?

Reply:
it used to be prefix till I demoed it, will update